app/assets/stylesheets/pages/idea.sass

```diff
@@ -18,11 +18,17 @@
   .idea-title
     min-height: 100px
   .idea-actions
+    display: flex
```

Comment:
Any explanation as to what this is for? Keep in mind that if we can avoid it, would rather use something else since support for it is pretty limited on most browsers that teachers will be using: http://www.w3schools.com/css/css3_flexbox.asp

E.g. it only supports IE 11 & up.